packages/frontend/src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -195,9 +199,28 @@ class SetupRecoveryMethod extends Component {
             await validateSecurityCode(accountId, method, securityCode);
             await saveAccount(accountId, recoveryKeyPair);
 
+            // IDENTITY VERIFIED FUNDED ACCOUNT
+            if (DISABLE_CREATE_ACCOUNT && !fundingOptions && ENABLE_IDENTITY_VERIFIED_ACCOUNT) {
+                try {
+                    // Call function directly to handle error silently
+                    await wallet.createIdentityFundedAccount({
+                        accountId,
+                        kind: method.kind,
+                        publicKey: recoveryKeyPair.publicKey,
+                        identityKey: method.detail,
+                        verificationCode: securityCode,
+                        recoveryMethod: method.kind
+                    });
+                } catch(e) {
+                    console.warn(e.code);
+                    await fundCreateAccount(accountId, recoveryKeyPair, method.kind);
+                }
+                return;
+            }
+
             // IMPLICIT ACCOUNT
             if (DISABLE_CREATE_ACCOUNT && !fundingOptions && !recaptchaToken) {
-                await fundCreateAccount(accountId, recoveryKeyPair, fundingOptions, method);
+                await fundCreateAccount(accountId, recoveryKeyPair, method.kind);
```

Comment:
`fundingOptions` was only passed in incorrectly inside `SetupRecoveryMethod.js` which handles email/phone recovery method account creation only. This resulted in `fundingOptions` taking the place of `recoveryMethod` since `fundCreateAccount` only takes three arguments. And since no `fundingOptions` were provided in this case, `fundingOptions === undefined`. 

`recoveryMethod` only becomes relevant down the line in `addLocalKeyAndFinishSetup`, after the account has been created, and it's only checking for `ledger` or `phrase` `recoveryMethods`, so the fallback with `recoveryMethod` as `undefined` wouldn't cause any bugs.